src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,131 @@
+package duke;
+import duke.commands.AddDeadline;
```

Comment:
Should there be a line break after the package statement before the import statements?